kodilla-advanced-tests/src/main/java/com/kodilla/mockito/homework/WeatherAlertsService.java

```diff
@@ -10,7 +10,7 @@
 
     private Map<String, Set<Subscriber>> weatherAlertService = new HashMap<>();
 
-    public void addSubscriber(String location, Set<Subscriber> subscribers ) {
+    public void addSubscriberToLocation(String location, Set<Subscriber> subscribers ) {
```

Comment:
wciaz nie to

Reply:
to co masz na myśli? nie umiem czytać w myślach, czy po skrótach myślowych